src/packages/cli/package.json

```diff
@@ -103,6 +105,7 @@
     "pg": "8.3.3",
     "pkg": "4.4.9",
     "pkg-up": "3.1.0",
+    "prettier": "^2.1.1",
```

Comment:
My commits were failing via pre-commit hook because I didn't have this globally installed.

Seems to me collaboration is easier by using a local prettier version.

Reply:
Yep, that's great!